modules/oadp-using-enable-api-group-versions.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: PROCEDURE
+[id="oadp-using-enable-api-group-versions_{context}"]
+= Working with different Kubernetes API versions in the same OADP operation
+
+By default, Velero only backs up resources that use the preferred version of the Kubernetes API. However, Velero also includes a feature, link:https://velero.io/docs/v1.9/enable-api-group-versions-feature/[Enable API Group Versions], that overcomes this limitation. When enabled on the source cluster, this feature causes Velero to back up _all_ Kubernetes API group versions that are supported on the cluster, not only the preferred one. Once the versions are stored in the backup tarball file, they are available to be restored on the destination cluster.
```

Comment:
> ! tarball noun
Do not use. Use ".tar file".

> ! once adverb
Use only to mean "one time". Do not use as a conjunction to mean "after" or "when".

Reply:
Done